README.md

```diff
@@ -24,7 +24,7 @@ docker build . -t ndt-server
 prepare the runtime environment
 
 ```bash
-install -d certs data
+install -d certs datadir
```

Comment:
I changed the name because `data` now contains Go code. It may be bad to suggest storing data with a default command line in a place where there is also Go code, because one may be tempted to `rm -rf` it.